package.json

```
@@ -9,6 +9,28 @@
     "ui"
   ],
   "resolutions": {
```

```suggestion
  "resolutions-note": "work-around for https://github.com/Agoric/agoric-sdk/issues/8621",
  "resolutions": {
```

also, to get a back-link from that issue:
 - https://github.com/Agoric/agoric-sdk/issues/8621